src/test/java/org/fundacionjala/coding/daniel/cipher/CipherTest.java

```diff
@@ -0,0 +1,94 @@
+package org.fundacionjala.coding.daniel.cipher;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for testing Caesar/Vigenere cipher methods.
+ */
+public class CipherTest {
+
+    private static final String KEY = "LIMON";
+    private CaesarCipher caesarCipher;
+    private VigenereCipher vigenereCipher;
+
+    /**
+     * Initializes variables.
+     */
+    @Before
+    public void setUp() {
+        final int caesarKey = 3;
+        caesarCipher = new CaesarCipher(caesarKey);
+        vigenereCipher = new VigenereCipher(KEY);
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class